src/libraries/Microsoft.PowerFx.Interpreter/Values/LambdaFormulaValue.cs

```diff
@@ -43,5 +43,11 @@ public override void Visit(IValueVisitor visitor)
         {
             return _tree.Accept(visitor, context);
         }
+
+        public override string ToExpression()
+        {
+            // !JYL! Do we need to serialize this?
+            throw new NotImplementedException("LambdaFormulaValue cannot be serialized.");
```

Comment:
>throw new NotImplementedException("LambdaFormulaValue cannot be serialized."); [](http://example.com/codeflow?start=12&length=78)

We probably should

Reply:
this one is tricky.... we can actually probably punt on it for now. 

This would be like arg1 passed into a filter - a lambda callback. 
Since those can't get passed around like regular values, the need to serialize is mitigated.

Reply:
Yeah, this is a hack for the interpreter, and can't meaningfully show up in a place where the host would want to serialize it. notice that this type is also `internal` unlike the other FormulaValues

Reply:
Nit - I'd expand the comment here to explain why we don't impl this.